docs/content/tutorial/step_15.ngdoc

```diff
@@ -0,0 +1,327 @@
+
+In order to update the information correctly, it is neccesary to add a _`debounce`_ option to the model.
+
+Please note that now the inputs are associated with theirs labels using `label` elements.
+
+The directives are based on an idea of [Marcy Sutton](http://marcysutton.com/slides/fluent2015/) about how to add accessibility in the AngularJS applications.
+
+
+### How to use the directives
+
+<br />
+**`app/phone-list/phone-list-template.html`:**
+
+```html
+
+  ...
+  
+  <div>
+    <label for="order">Sort by:</label>
+    <select inform-order id="order" ng-model="$ctrl.orderProp">
+      <option value="name">Alphabetical</option>
+      <option value="age">Newest</option>
+    </select>
+  </div>
+
+  ...
+
+  <ul result-list class="phones">
+    <li ng-repeat="phone in filtered = ($ctrl.phones | filter:$ctrl.query | orderBy:$ctrl.orderProp)"
+        class="thumbnail phone-list-item">
+      <a href="#!/phones/{{phone.id}}" class="thumb">
+        <img ng-src="{{phone.imageUrl}}" alt="{{phone.name}}" />
+      </a>
+      <a href="#!/phones/{{phone.id}}">{{phone.name}}</a>
+      <p>{{phone.snippet}}</p>
+    </li>
+  </ul>
+
+```
+
+<br />
+**`app/index.html`:**
+
+```html
+  ...
+  <script src="phone-list/phone-list.directive.js"></script>
+  ...
+```
+
+
+<br />
+**`app/phone-list/phone-list.directive.js`:**
+
+```js
+
+'use strict';
+
+/* Directives */
+
+angular.
+  module('phoneList').
+  directive('resultList', [function () {
+    var ariaStatus = document.querySelector('.aria-status');
+    return {
+      restrict: 'A',
+      link: function ($scope) {
+        $scope.$watch('filtered.length', function (length) {
+          if(length === 1) {
+            ariaStatus.innerHTML = length + ' item found';
+          } else if(length > 1) {
+            ariaStatus.innerHTML = length + ' items found';
+          } else {
+            ariaStatus.innerHTML = 'No items found';
+          }
+        });
+      }
+    }
+}
+]).
+  directive('informOrder', [function () {
+      var ariaStatusSort = document.querySelector('.aria-status-sort');
+      return {
+        restrict: 'A',
+        link: function ($scope) {
+          $scope.$watch('$ctrl.orderProp', function (order) {
+            if(order === 'age') {
+              ariaStatusSort.innerHTML = 'Items filter by newest'
+            } else {
+              ariaStatusSort.innerHTML = 'Items filter by alphabetical order'
+            }
+            
+          });
+        }
+      }
+  }
+]);
+
+```
+
+## Using the keyboard
+
+The current detail of a phone prevents a user to use a keyboard to navigate for over each thumbnail image. It is very important due to accessibility that all the actions and information are available via keyboard too. For example, lightboxes, carousels, slides, etc.
+
+_Tip_: Try to navigate over the application using only the _TAB_ key. You will discover that some actions can't be accessed nor the focus on some elements. 
```

Comment:
I've added how to fix this problem (adding a button element). For this reason, and after check again the project, the `ngAria` module is not necessary (I added it, to activate the `ng-click` inside the `img` element).